backend/WebApi/Helpers/InputValidator.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Domain.Entities;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebApi.NewFolder
```

Comment:
I guess this should be `WebApi.Helpers`?

Reply:
I just auto implemented it I don't know why it did it like that

Reply:
Gonna check it out